misc/convertzone/main.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+/*
+convertzone: Read BIND-style zonefile and output.
+
+     convertzone [-mode=MODE] zonename [filename]
+
+     -mode=tsv   TAB-separated values (default)
+     -mode=dsl   DNSControl DSL
+     -mode=pretty   Sort and pretty-print records.
+
+     zonename    The FQDN of the zone name.
+     filename    File to read (default: stdin)
+*/
+
+import (
+	"bufio"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/StackExchange/dnscontrol/providers/bind"
+	"github.com/miekg/dns"
+	"github.com/miekg/dns/dnsutil"
+)
+
+var output_mode = flag.String("mode", "tsv", "tsv|dsl|pretty")
+var flag_defaultTtl = flag.Uint("ttl", 300, "Default TTL")
```

Comment:
lint is throwing fits about underscore variable names. lets nip this in the bud and use camelCase for Go variables.